index.html

```diff
@@ -186,6 +186,20 @@ <h2>Scope</h2>
           respectively.
         </p>
 
+        <p>
+          Depending on community discussions and demands, this Working Group may also decide to submit the EPUB 3.3, 
+          the EPUB Reading Systems 3.3, and the EPUB Accessibility 1.1 specifications to 
+          <a href="https://www.iso.org/committee/45020.html">ISO/IEC JTC1</a> to be published as ISO/IEC standards.
+          This submission would follow the <a href="https://www.w3.org/2010/03/w3c-pas-submission.html">PAS Submission process</a>.
+          This would obsolete the current ISO/IEC 23736-1-6:2020 standards based on 
+          <a href="https://idpf.org/epub/301/spec/epub-publications.html">EPUB 3.01</a> (originally published in 2014), 
+          as well as the separate ISO/IEC 23761:2021 EPUB Accessibility standard.
+          The PAS submission would not affect the current distribution rights of the documents, which will remain free on
+          www.w3.org, and it can also be expected that the documents will be available on the ISO Web site at no costs as
+          <a href="http://standards.iso.org/ittf/PubliclyAvailableStandards/index.html">JTC1 freely available standards</a>.
+          The submission would only require minor editorial changes on the versions of the specifications submitted to ISO.
```

Comment:
suggest s/require/permit/

Reply:
+1, although... the term "require" is from ISO's point of view, whereas the term "permit" is from W3C's point of view. Ie, these are not exactly the same, but I am fine to emphasize the W3C point of view, so I am doing the change.